<ui version="4.0" >
 <class>SearchWindowBase</class>
 <widget class="QDialog" name="SearchWindowBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SearchWindowBase</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>55</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Text:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="textComboBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>55</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="directoryComboBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton" >
       <property name="text" >
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFilesTable" name="filesTable" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="settingsButton" >
       <property name="text" >
        <string>Se&amp;ttings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="journalButton" >
       <property name="text" >
        <string>&amp;Journal</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>101</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="findButton" >
       <property name="text" >
        <string>&amp;Find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton" >
       <property name="text" >
        <string>&amp;Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFilesTable</class>
   <extends>QTableWidget</extends>
   <header>qfilestable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
